apps/data-service-catalog/app/catalogs/[catalogId]/data-services/data-services-page-client.tsx

```diff
@@ -60,36 +107,130 @@ const DataServicesPageClient = ({ dataServices, catalogId, hasWritePermission }:
+              <PlusCircleIcon />
+              {localization.dataServiceCatalog.button.newDataService}
+            </LinkButton>
+          </SearchHitsPageLayout.ButtonRow>
+        )}
         <SearchHitsPageLayout.LeftColumn>
-          <Paragraph>{`${localization.add}...`}</Paragraph>
-          <StatusFilter onStatusChange={handleStatusChange} />
+          <Search
+            variant='primary'
+            placeholder={localization.search.search}
+            onSearchClick={handleSearch}
+            onKeyDown={handleSearchKeyDown}
+          />
+          <FilterChips />
+          <SearchFilter distributionStatuses={distributionStatuses} />
         </SearchHitsPageLayout.LeftColumn>
         <SearchHitsPageLayout.MainColumn>
-          <SearchHitTable dataServices={filteredDataServices} />
+          <SearchHitContainer
+            searchHits={
+              filteredDataServices.length > 0
+                ? filteredDataServices.map((dataService: DataService) => (
+                    <div
+                      key={dataService.id}
+                      className={styles.searchHit}
+                    >
+                      <SearchHit
+                        title={getTranslateText(dataService?.title)}
+                        description={getTranslateText(dataService?.description)}
+                        titleHref={`/catalogs/${catalogId}/data-services/${dataService?.id}/edit`}
+                        statusTag={
+                          dataService.status && (
+                            <Tag.DataServiceStatus
+                              statusKey={
+                                findDistributionStatus(dataService.status)
+                                  ?.code as DataServiceStatusTagProps['statusKey']
+                              }
+                              statusLabel={
+                                getTranslateText(findDistributionStatus(dataService.status)?.label) as string
+                              }
+                            />
+                          )
+                        }
+                        content={
+                          <>
+                            <div className={styles.set}>
+                              <p>
+                                {localization.lastChanged} {formatDate(dateStringToDate(dataService.modified ?? ''))}
+                              </p>
+                              <span>•</span>
+                              {dataService.published
+                                ? localization.publicationState.publishedInFDK
+                                : localization.publicationState.unpublished}
+                            </div>
+                          </>
+                        }
+                      />
+                    </div>
+                  ))
+                : null
+            }
+            noSearchHits={!filteredDataServices || filteredDataServices.length === 0}
+          />
```

Comment:
Ingen paginering og server side sortering?

Reply:
Nope, det har vi kun for begrep. Å gå over til det for de andre katalogene tenker jeg fint kan vente til etter overgang til catalog-frontend